utils/docker/peer/Dockerfile

```diff
@@ -4,11 +4,11 @@
 
 FROM hyperledger/fabric-private-chaincode-base:latest
 
-ARG GO_VERSION=go1.13
+ARG GO_VERSION=go1.14
```

Comment:
interesting. This doesn't match the 1.11 earlier. Is fabric 2.0 or 2.0.1 at 1.14? They usually aren't so aggressive in upgrading.

Reply:
see comment above